### DISCUSSION TITLE
Supported PostgreSQL Version

### DISCUSSION THREAD

**Discussion by SyamprasadNakka** [apache/superset#14311] [Q&A / Help]
Hello,

We are running **"apache-superset:0.36"** version and it's connected to _**PostgreSQL-11**_ server. 
We would like to upgrade the Postgres database to **_PostgreSQL -13_** version and want to know whether it's supported version or not for superset.

Can you please suggest the inputs or provide if there is any version matrix document.

Regards,
Syam

**Comment by villebro** [comment]
(converted to Q&A discussion, as this is not an issue) 
Hi @SyamprasadNakka ,

If you're referring to the database that's used to store the metadata for Superset, I don't believe upgrading from v.11 to v.13 should be an issue.

**Reply by SyamprasadNakka** [reply]
Do we have any superset documentation to share it with my team.

Regards,
Syam
